pixel/settings.py

```diff
@@ -24,6 +24,8 @@ class Base(Configuration):
 
     DEBUG = False
 
+    ADMINS = [('Julien Maupetit', 'julien@tailordev.fr'), ]
```

Comment:
I think a generic email address like: `pixel@candihub.eu` that redirects to @gaellelelandais @biorosetics and myself (during the development phase) must be preferred.

WDYT @gaellelelandais & @biorosetics?

Reply:
:+1:

Reply:
I agree. A generic adress is a good idea.